packages/react-native/React/Views/RCTFont.h

```diff
@@ -19,6 +20,7 @@ typedef UIFont * (^RCTFontHandler)(CGFloat fontSize, NSString *fontWeightDescrip
  */
 RCT_EXTERN void RCTSetDefaultFontHandler(RCTFontHandler handler);
 RCT_EXTERN BOOL RCTHasFontHandlerSet(void);
+RCT_EXTERN RCTFontWeight weightOfFont(UIFont *font);
```

Comment:
Since it's exported now, we should namespace it.

`RCTGetFontWeight`

Reply:
@javache Done.